api/backend/application/graph_service.py

```
@@ -144,6 +142,17 @@ async def graph_from_poly(city_id, polygon):
     access_nodes = list(map(access_node_obj_to_list, access_nodes_raw))
     access_edges = list(map(access_edge_obj_to_list, access_edges_raw))
 
+    points, edges, access_nodes, access_edges, node_mapping = merge_close_nodes(
```

Ruff (0.14.8)

Unpacked variable `node_mapping` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)